HeterogeneousCore/AlpakaTest/plugins/TestAlpakaAnalyzer.cc

```diff
@@ -134,19 +139,32 @@ class TestAlpakaAnalyzer : public edm::global::EDAnalyzer<> {
       assert(vi.id() == i);
       assert(vi.m() == matrix * i);
     }
+
+    if (expectBackend_) {
+      auto const& backend = event.get(backendToken_);
+      if (*expectBackend_ != backend) {
+        throw cms::Exception("Assert") << "Expected input backend " << *expectBackend_ << ", got " << backend;
```

Comment:
I think it would be more informative to print the string representation of the backend, rather than the number.

Reply:
Ah OK, these are just tests, so it doesn't really matter.

Reply:
Would it be otherwise useful to be able to get the string representation of the backend? I had such function at some point during development, but left it out from the PR because it wasn't strictly needed (beyond test code). Would e.g. some DQM code benefit from it (like for histogram bin labels)?

Reply:
String representation is now used in the printout.